### PR TITLE
SBC_COMMON : Redirect the user to create account if the user doesn't have valid role#3296

### PR DESCRIPTION
**Pull request by nitheesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/3296

*Description of changes:*
SBC_COMMON : Redirect the user to create account if the user doesn't have valid role

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/NonBcscAccounts/AffidavitDownload.vue [hunk 1]
```diff
@@ -73,37 +73,64 @@
...
-      <v-btn
-        large
-        color="primary"
-        class="font-weight-bold"
-        data-test="register-bceid-btn"
-        @click="redirectToBceId"
-      >
-        Register a new BCeID
-      </v-btn>
+      <template v-if="isAuthenticated">
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/NonBcscAccounts/AffidavitDownload.vue:76]: is this change related to this ticket?

Comment (saravanpa-aot) [reply auth-web/src/components/auth/NonBcscAccounts/AffidavitDownload.vue:76]: the changes looks like if isAuthenticated , then show the signin buttons ? a bit confused here..could be a correct change , just wants to understand it more

Comment (nitheesh-aot) [reply auth-web/src/components/auth/NonBcscAccounts/AffidavitDownload.vue:76]: Yes, its related to this.
actually if isAuthenticated, then we show the Continue button, otherwise the signin/register buttons
